compact.go

```diff
@@ -475,6 +477,10 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 		return errors.Wrap(err, "write merged meta")
 	}
 
+	// We are explicitly closing them here to check for error even
+	// though these are covered under defer. This is because in Windows,
+	// you cannot delete these unless there is they are closed and we want to close them
+	// in case of any errors above.
```

Comment:
> This is because in Windows,
you cannot delete these unless there is they are closed and we want to close them
in case of any errors above.

I don't understand this comment